galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/TestRunner.java

```diff
@@ -412,7 +412,7 @@ public void runTest(Properties bootstrapProperties, Properties overridePropertie
             }
         } else if (this.runType == RunType.SharedEnvironmentBuild) {
             recordCPSProperties(frameworkInitialisation);
-            updateStatus("up", "built");
+            updateStatus(TestRunLifecycleStatus.UP, "built");
```

Comment:
If the status is 'built' why are we calling it 'UP' in the constant ? Maybe Im missing something...

Reply:
To be potentially changed in another issue.